source/georef/street_network.cpp

```diff
@@ -389,16 +390,53 @@ PathFinder::start_dijkstra_and_fill_duration_map(const navitia::time_duration& r
     return result;
 }
 
+navitia::time_duration
+PathFinder::exclude_sp_with_free_radius_filter(const type::GeographicalCoord& sp,
+                                               const std::vector< std::pair<idx_t, type::GeographicalCoord> >& excluded_sp_list,
+                                               const  navitia::time_duration& duration) const {
+    auto it = std::find_if(excluded_sp_list.begin(), excluded_sp_list.end(),
+                           [&sp](const std::pair<idx_t, type::GeographicalCoord>& input)
+                           {return input.second == sp;} );
+    // if it matches, sp duration = 0
+    if (it != excluded_sp_list.end()) {
+        LOG4CPLUS_DEBUG(log4cplus::Logger::getInstance(LOG4CPLUS_TEXT("logger")),
```

Comment:
it's minor but for me this is a trace while your log at https://github.com/CanalTP/navitia/pull/2386/files#diff-04efa2f2956b5a07b2fef399f54021f7R431 is more a debug.

Reply:
Okay, I take note of that remark